### DISCUSSION TITLE
Visit as a geometry, result viewer ?

### DISCUSSION THREAD

**Discussion by PinkMushroom** [thliebig/openEMS-Project#68] [General]
In my travels through the web, I ran across this 

https://visit-dav.github.io/visit-website/

and it looks quite nice !

I've got it installed and I was going to maybe try and see about using it as both a geometry viewer and a (3D field) result viewer.

I'm starting with the geometry view looking at the 3D file formats listed on this page for OpenEMS,

https://wiki.openems.de/index.php/AppCSXCAD.html

Visit can handle a LOT of formats but in particular there is an xdmf (which is XML based) format which looks like it might be very similar to the x3d format CSXCAD can generate

There is also a polydata-VTK is also listed which would probably work too.

I was looking for suggestions on what might be the best format to try and use, for example are one of the formats listed used more, or are better maintained ?

Thank you.

**Comment by thliebig** [comment]
AppCSXCAD is based on vtk. Therefore everything vtk can export should be a good choice. I would not know which is best. I think this heavily depends on what you want to do. For example export the entire scene or just individual objects and so on.
